CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [0.4.10] - SNAPSHOT
+### Added
```

Comment:
This is no longer a 0.4.x release

Reply:
Please leave 0.4.10 stuff there.

Please create a new [0.5.0] section and add this there.

This, AS-IS, will have conflicts that will need to be resolved with 0.4.10.

Reply:
👍 done